dataprofiler/profilers/profiler_options.py

```diff
@@ -1170,6 +1200,7 @@ def __init__(self, null_values=None):
         self.data_labeler = DataLabelerOptions()
         self.correlation = CorrelationOptions()
         self.chi2_homogeneity = BooleanOption(is_enabled=True)
+        self.synthetic_data = SyntheticDataOptions()
```

Comment:
update to `StructuredOptions` I think

Reply:
This options is an attribute of the `StructuredOptions` class (the code highlighted is in the `def __init__` of the `StructuredOptions` class) so it wouldn’t really make sense to use `StructuredOptions` here. Was thinking of a simple `BooleanOptions`.